### ISSUE TITLE
Homework 1 repository cannot be found

### ISSUE THREAD

**Issue by transcendraz** [CSCI201-Fall2018/homework-1-IDRALORAS-USC#1]
Hello:
I am Chen Tang, currently learning csci201. However, the first homework says I received an zero because can’t find my git repository.
I feel confused because I have uploaded it 10 days ago. I am sure there are some mistakes in here that I hope we can solve together.

Best regards 
Chen Tang.
<img width="1407" alt="screen shot 2018-09-09 at 11 36 08 am" src="https://user-images.githubusercontent.com/31268914/45267648-39b8c480-b425-11e8-85c8-29dfcd747800.png">

**Comment by transcendraz** [comment]
Hi:
I think I figured out, it is because my repo is set to private.
I am sorry to make that mistake, and I hope I can receive another grading, because I indeed finished it way before due, and it is just a minor mistake.
The GitHub indicates my updates are done more than 10 days ago, so it proves I didn’t make changes to the file I uploaded. 
This message is sent to professor Jeffery and two TA, because I don’t know exactly who to find with this issue. 
Sorry for bothering your time.

Best regards
Chen Tang
